content/gherkin/reference.md

```diff
@@ -330,6 +331,60 @@ to match the step against a step definition.
 
 You can also use parameters in [multiline step arguments](#step-arguments).
 
+## Rule
+
+The `Rule` entity is a rather new idiom in Gherkin (since Gherkin v6).
```

Comment:
Suggestion: "As of Gherkin 6, a new keyword `Rule` has been added." 

**Note:** I'm not sure Gherkin 6 has been implemented for all languages yet. Afaik it is still pending for Cucumber-jvm (and I don't use Js / Ruby personally).

Reply:
You basically have a composition problem. Composition problems often do not scale.
The more Cucumber implementations you get, the worse you are. Meaning, one implementation may block you forever.

A better approach would be to document that the Javascript implementation describes which Gherkin version it supports (or there is a table for all implementations).

MAYBE: @aslakhellesoy and @mattwynne should probably provide some feedback what is best.

Otherwise, decide what you think is best.